src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
```

Comment:
[CODE KNOWLEDGE] - Instead of using the `function` keyword, use the method shorthand in object literals. For example, use `add(num) { ... }` instead of `add: function(num) { ... }`.